src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/repository/TaskResourceRepositoryTest.java

```diff
@@ -40,10 +41,15 @@ void setUp() {
 
     @Test
     void shouldReadTaskData() {
-
-        createAndSaveTask();
+        String taskId = "8d6cc5cf-c973-11eb-bdba-0242ac11001e";
+        createAndSaveTask(taskId);
```

Comment:
Note a suggestion but rather a note: create and save task should be able to return you the object it created as a TaskResource, you could pull the id from there too.

Reply:
resolved